React table throws Can't resolve lodash-es in v2.11.1

I'm getting the following error after updating to latest react-tables, v2.11.1.

```
Failed to compile.

./node_modules/@patternfly/react-table/dist/esm/components/Table/base/body.js
Module not found: Can't resolve 'lodash-es' in '/home/spytec/Documents/JavaScriptProjects/telltale-ui/node_modules/@patternfly/react-table/dist/esm/components/Table/base'
```

--- a/packages/patternfly-4/react-table/package.json
+++ b/packages/patternfly-4/react-table/package.json
@@ -33,10 +33,10 @@
     "@patternfly/react-styles": "^3.3.3",
     "@patternfly/react-tokens": "^2.5.5",
     "classnames": "^2.2.5",
-    "exenv": "^1.2.2"
+    "exenv": "^1.2.2",
+    "lodash-es": "^4.2.1"
   },
   "peerDependencies": {
-    "lodash-es": "4.x",
     "prop-types": "^15.6.1",
     "react": "^16.4.0",
     "react-dom": "^15.6.2 || ^16.4.0"

